check-dnt.js

```diff
@@ -0,0 +1,12 @@
+function toggleStatus(enabled) {
+    var status = window.document.getElementById("dnt-" + enabled);
+    var waiting = window.document.getElementById("dnt-waiting");
+    waiting.classList.add("hidden");
+    status.classList.remove("hidden");
+}
+
+if (navigator.doNotTrack == "yes") {
```

Comment:
You're right, I'm testing this with Nightly and I see "1" now. I will check for both.